MidasCivil_Adapter/Convert/ToBHoM/Results/ToBarStress.cs

```diff
@@ -65,6 +67,47 @@ public static BarStress ToBarStress(this List<string> delimitted, string forceUn
 
         /***************************************************/
 
+        public static BarStress ToBarStress(List<object> item)
+        {
+            double position = GetBarResultPosition(item[3].ToString());
+
+            List<double> combBends = new List<double> { 
+                System.Convert.ToDouble(item[11].ToString()), 
+                System.Convert.ToDouble(item[12].ToString()), 
+                System.Convert.ToDouble(item[13].ToString()), 
+                System.Convert.ToDouble(item[14].ToString()) 
+            };
+            
+            double combBendPos = combBends.Max();
+            double combBendNeg = combBends.Min();
+
+            //TODO: resolve below identifiers extractable through the API
+            int mode = -1;
+            double timeStep = 0;
+            int divisions = 0;
```

Comment:
```suggestion
            int divisions = 0;
```
The default for MidasCivil is 4, I don't think this can be changed.